bedrock/products/templates/products/vpn/platforms/desktop.html

```diff
@@ -0,0 +1,173 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "products/vpn/includes/macros.html" import hero_image, join_waitlist_button, vpn_nav_cta, vpn_content_well with context %}
+
+{% extends "products/vpn/base.html" %}
+
+{% block page_title_full %}Mozilla VPN: Protect Your Entire Device{% endblock %}
+{% block page_title_suffix %}{% endblock %}
+
+{% block page_desc %}Use the Mozilla VPN for full-device protection for all apps. With servers in 30+ countries, you can connect to anywhere, from anywhere.{% endblock %}
+
+{% block body_id %}mozilla-vpn-landing{% endblock %}
+
+{% set _utm_source = 'www.mozilla.org-vpn-product-page' %}
+{% set _utm_campaign = 'vpn-product-page' %}
+{% set _params = '?utm_source=' ~ _utm_source ~ '&utm_medium=referral&utm_campaign=' ~ _utm_campaign %}
+
+{% block page_css %}
+  {{ css_bundle('mozilla-vpn-platforms') }}
+{% endblock %}
+
+{% block site_header %}
+  {% with
+    custom_nav_cta=vpn_nav_cta(
+      link_text='Try Mozilla VPN',
+      alt_link_text='Join the Waitlist',
+      sign_in_link_text='Sign In',
+      utm_source=_utm_source,
+      utm_campaign=_utm_campaign,
+    )
+  %}
+    {% include 'includes/protocol/navigation/navigation.html' %}
+  {% endwith %}
+{% endblock %}
+
+{% block content %}
+<main>
+  <section class="vpn-hero">
+    <div class="mzp-l-content mzp-t-content-xl">
+      <div class="vpn-hero-container">
+        <div class="vpn-hero-body">
+          <h1 class="vpn-hero-title">Mozilla VPN 2</h1>
+          <h2 class="vpn-hero-subtitle">{{ ftl('vpn-desktop-hero-headline') }}</h2>
+          <div class="vpn-hero-desc">
+            <p>{{ ftl('vpn-desktop-hero-copy') }}</p>
+
+            <a class="vpn-button" href="{{ url('products.vpn.landing') }}" data-link-name="Mozilla VPN">{{ ftl('vpn-shared-cta') }}</a>
+
+            {{ join_waitlist_button(link_text='Join the Waitlist', position='primary') }}
+
+            <div class="vpn-hero-guarantee">
+                <em>{{ ftl('vpn-shared-day-guarantee') }}</em>
+            </div>
+          </div>
+        </div>
+        <div class="vpn-hero-image hide-from-legacy-ie" data-illustration="n-1">
+          {{ hero_image(
+            'img/products/vpn/landing/vpn-hero-01.png',
+            'img/products/vpn/landing/vpn-hero-01-high-res.png',
+            'n-1')
+          }}
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <div class="mzp-l-content mzp-t-content-xl">
+
+    {% call vpn_content_well(
```

Comment:
Just a callout: there's a fair chance these macros might change next week, at least as far as styling and spacing of the component goes. I can't say at this stage how much effect it may have on these pages, but appearence will at least change a little.

Reply:
Should I copy/fork this function?

Reply:
I really don't know what's best here honestly. I'd probably suggest forking everything for this page until the dust settles.

Reply:
Duplicated!